tensorboard/defs/vulcanize.bzl

```diff
@@ -27,6 +27,10 @@ def _tensorboard_html_binary(ctx):
   script elements (sources inside element and content of JavaScript src they
   point at). The hashes are delimited by newline.
   """
+
+  if ctx.attr.extract_script and not ctx.attr.script_path:
+    fail("script_path is required when extract_script is True.")
```

Comment:
Conversely, it doesn’t make sense for `script_path` to be provided
unless `extract_script` is set, right? Maybe consider just letting
`extract_script` or `output_script` be the output path rather than
requiring two attributes to be in sync:

```python
tensorboard_html_binary(
    name = "index",
    compile = True,
    input_path = "/tensorboard.html",
    output_path = "/index.html",
    output_script = "/index.js",
    deps = [":tensorboard"],
)
```